routers/common/middleware.go

```diff
@@ -107,8 +107,17 @@ func ForwardedHeadersHandler(limit int, trustedProxies []string) func(h http.Han
 	return proxy.ForwardedHeaders(opt)
 }
 
-func Sessioner() func(next http.Handler) http.Handler {
-	return session.Sessioner(session.Options{
+func Sessioner() (middleware func(next http.Handler) http.Handler, err error) {
+	// Recover from panic if session.Sessioner fails due to invalid config
```

Comment:
Maybe invoke
```go

opt := PrepareOptions(options)

manager, err := NewManager(opt.Provider, opt)
if err != nil {
return nil, err
}
```
instead of catch a panic.